pipeline/deploy.sh

```diff
@@ -15,31 +15,102 @@ then
 fi
 
 # Borrow a testbed from CSI Testbed Pool Svc.
-if ! testbed=$(curl -X 'GET' "${CNS_TESTBEDPOOL_SVC_URL}");
+if ! testbed=$(curl -X 'PUT' "${CNS_TESTBEDPOOL_SVC_URL}" -H 'accept: application/json' -u "${CNS_MANAGER_USERNAME}:${CNS_MANAGER_PASSWORD}");
 then
 	echo "Unable to borrow a testbed!"
 	exit 1
 fi
+
 echo "TestbedInfo: $testbed"
 
+# Extract testbed ID from borrow testbed API Response
```

Comment:
Can you end the statement with a period? This applies to all the comments in this file.